src/funnels/Swapper.sol

```diff
@@ -100,13 +101,17 @@ contract Swapper {
             limits[src][dst].zzz = limit.zzz;
         }
 
-        uint256 prevDstBalance = GemLike(dst).balanceOf(buffer);
+        uint256 prevBalance = GemLike(dst).balanceOf(address(this));
```

Comment:
Do we really care about the prev balance thinking we are now checking the swapper? Even if someone donated the dst token to the Swapper I wouldn't think it is a big deal.

Reply:
Removed it.